src/content/docs/en/guides/typescript.mdx

```diff
@@ -122,11 +122,11 @@ import Layout from '@layouts/Layout.astro';
 You may want to add a property to the global object. You can do this by adding top-level declarations using the `declare` keyword to your `env.d.ts` file:
```

Comment:
```suggestion
You may want to add a property to the global object. You can do this by adding top-level declarations using `declare var` in your `env.d.ts` file:
```
So, what do you think about just mentioning here so that we don't need a whole other sentence?

Reply:
In my opinion this change doesn't fit, because it is also possible to declare a function with `declare function`. With this change it becomes to specific.

We should leave this sentence as it is.

Reply:
Ah, great call out! OK, then yeah, change doesn't work.

So, we'll let the example do the speaking. At least people won't be led astray by a faulty example, and they will tend to copy the example.